src/reducers/isMenuOpened.js

```diff
@@ -0,0 +1,14 @@
+import { TOGGLE_MENU_OPENED, SET_MENU_OPENED } from "../actions";
+
+const isMenuOpened = (state = false, action) => {
```

Comment:
This name is a little confusing. Maybe rename this reducer.

Reply:
And I think that the state should be an object with properties.
```
const initMenuReducerState = {
  isOpen: false
};

...

const MenuReducer = (state = initMenuReducerState, action) => {
...
```

Reply:
Well, since we are combining reducers using `combineReducers` the name of the reducer is maped directly onto a prop in state. In that case, considering name `isMenuOpened` as a state property, then the name is as straightforward as it gets, I suppose.

Also, as the state should be immutable, storing objects in state, when not needed now, is extra hustle for us - I would prefer to keep it as granular as possible.
https://redux.js.org/recipes/structuringreducers/immutableupdatepatterns

Reply:
maybe isMenuOpenedReducer?